### PR TITLE
upstream fonts.css does not have mapping to 'common'

### PR DESCRIPTION
**Pull request by jvonau**
### Fixes bug:
noted error in https://github.com/iiab/iiab/pull/3196#issuecomment-1112452495

### CODE REVIEW

File: roles/osm-vector-maps/tasks/install.yml [hunk 1]
```diff
@@ -186,6 +185,13 @@
   with_fileglob:
     - fonts/*
 
+- name: Force Symlink {{ doc_root }}/common/fonts/fonts.css -> {{ vector_map_path }}/viewer/assets/fonts.css
```
Comment (holta) [inline roles/osm-vector-maps/tasks/install.yml]: Do you mean...?

```suggestion
- name: Force Symlink {{ vector_map_path }}/viewer/assets/fonts.css -> {{ doc_root }}/common/fonts/fonts.css
```

Comment (jvonau) [reply roles/osm-vector-maps/tasks/install.yml]: yup backwards

**Comment by holta** [comment]
Good enough to merge?

**Comment by holta** [comment]
This seems low-risk and useful to push forward community testing.

**Comment by georgejhunt** [comment]
The fonts are still not being found. The text is with serif.  

I guess it was not low risk.

**Comment by holta** [comment]
> The fonts are still not being found. The text is with serif.
>
> I guess it was not low risk.

@jvonau can you confirm this PR should be reverted?

**Pull request merged**